### PR TITLE
Add baker set key to concordium client topic. Update transactions table.

### LINKED ISSUE #197: Document how to update baker keys with concordium-client
As the title states.

### PR DESCRIPTION
**Pull request by dg-concordium**
## Purpose

To inform the user how to use the baker set-keys transaction in Concordium client.

## Changes

Added section about updating baker keys. Other minor updates in the topic. Updated the concordium client transactions table with cross reference to the Become a baker using Concordium client topic.

## Checklist

- [X] My code follows the style of this project.
- [X] The code compiles without warnings.
- [X] I have performed a self-review of the changes.
- [ ] I have documented my code, in particular the intent of the
      hard-to-understand areas.
- [ ] (If necessary) I have updated the CHANGELOG.

## CLA acceptance

_Remove if not applicable.

By submitting the contribution I accept the terms and conditions of the
Contributor License Agreement v1.0
- link: https://developers.concordium.com/CLAs/Contributor-License-Agreement-v1.0.pdf

- [X] I accept the above linked CLA.

Closes #197

### CODE REVIEW

File: source/shared/net/guides/become-baker.rst [hunk 1]
```diff
@@ -300,6 +303,34 @@ Participating in the finalization committee produces rewards on each block that
...
+
+You can choose an arbitrary name for the ``keys file``.
+
+Then run the transaction:
+
+.. code-block:: console
+
+   $concordium-client baker set-key <keys-file>.json --sender <account> --out <concordium-data-dir>/baker-credentials.json
+
+``--sender`` is the name or address of the transaction's sender account. The name is the one that's used when you :ref:`import the account<concordium-client-import-accounts-keys>` (assuming that this
```
Comment (abizjak) [inline source/shared/net/guides/become-baker.rst:323]: Technically `sender` can also be an address. But maybe mentioning that would detract from the flow here?

File: source/shared/net/guides/become-baker.rst [hunk 2]
```diff
@@ -300,6 +303,34 @@ Participating in the finalization committee produces rewards on each block that
...
+was done). It defaults to the account name "default".
+
+If you want to keep the baker-credentials.json output file in the same location as your other Concordium files, you can omit ``<concordium-data-dir>/``.
```
Comment (abizjak) [inline source/shared/net/guides/become-baker.rst:326]:
```suggestion
If you want to keep the ``baker-credentials.json`` output file in the same location as your other Concordium files, you can omit ``<concordium-data-dir>/``.
```

**Review by abizjak — APPROVED** [review]
A couple of minor things, otherwise looks good.

File: source/shared/net/guides/become-baker.rst (hunk 1 above)
Comment (dg-concordium) [reply source/shared/net/guides/become-baker.rst:323]: Yeah, I think so. Let's get it out there and see if we need to clarify it.

**Pull request merged**